setup/index.js

```diff
@@ -1,7 +1,9 @@
 const core = require('@actions/core');
 const tc = require('@actions/tool-cache');
 const { Octokit } = require('@octokit/rest');
+const {createActionAuth} = require('@octokit/auth-action');
```

Comment:
This will use the GitHub Workflow provided GITHUB_TOKEN, which has much higher API rate limits than the current set up and makes it very unlikely to hit the rate limits